Generator/Generator/BuiltinGen.swift

```diff
@@ -601,8 +612,8 @@ func generateBuiltinClasses (values: [JGodotBuiltinClass], outputDir: String?) a
             if bc.name == "Callable" {
                 p ("/// Creates a Callable instance from a Swift function")
                 p ("/// - Parameter callback: the swift function that receives an array of Variant arguments, and returns an optional Variant")
-                p ("public init (_ callback: @escaping ([Variant])->Variant?)") {
-                    p ("content = CallableWrapper.makeCallable (callback)")
+                p ("public init(_ callback: @escaping (borrowing Arguments) -> Variant?)") {
+                    p ("content = CallableWrapper.callableVariantContent(wrapping: callback)")
                 }
```

Comment:
This breaks the existing API, is there a reason to not introduce this as an additional constructor/optimization instead of breaking existing code?

Reply:
The main reason for removing the older one is that 

```swift
let callable = Callable { args in 
    // ...
}
```

becomes ambiguous. Depending on how people accessed the old args (by not passing the lambda from somewhere else and just using indexing operator) it can be potentially not source-breaking

Reply:
I returned it back and marked it as deprecated. Do you think it's a viable approach?

Reply:
I think this works, I will try it with my big codebase now and report back, we might need to tune it in the future.